profiles/common.nix

```diff
@@ -8,6 +8,15 @@
     ./common/zfs.nix
   ];
 
+  # Use systemd during boot as well
+  boot.initrd.systemd.enable = true;
```

Comment:
We cannot do this as long hetzner exists...

Reply:
Do you remember what the root cause was?

Reply:
Hetzner does not allow you to boot machines with EFI. I think they now do this if you open a ticket but this is not a good out-of-the-box experience. And than we also have many vps provider that do support EFI as well.

Reply:
How is having systemd in the initrd linked to EFI?

Reply:
Ah. Sorry. I was referring to systemd-boot being enabled. Not the initrd option.
I think the only reason not to use this yet here is if you have `boot.initrd.network.enable` set i.e. if you try to enter your disk encryption password via ssh, simply because this is not supported yet.